testing/tests/DevExpress.ui.widgets/popup.tests.js

```diff
@@ -2907,11 +2925,24 @@ QUnit.module('positioning', {
                 assert.strictEqual(newPosition.top, position.top - 100, 'top coordinate is correct');
             });
 
+            QUnit.test('popup should have new position after on/off fullScreen and resize', function(assert) {
```

Comment:
Just updated to match the module name
```suggestion
            QUnit.test('on/off fullScreen and resize should not be restored', function(assert) {
```